src/components/controllers/tutorial/index.jsx

```diff
@@ -140,42 +143,38 @@ export function Tutorial({ html, meta }) {
 					orientation="horizontal"
 					force={!showCode ? '100%' : undefined}
 					other={
-						// TODO: CodeMirror v5 cannot load in Node, and loading only the runner
-						// causes some bad jumping/pop-in. For the moment, this is the best option
-						typeof window === 'undefined'
-							? null
-							: <Splitter
```

Comment:
There is already a `if (typeof window === 'undefined')` return above, this does nothing. Was a relic from when I was trying to see if I could prerender the content portion of the tutorial, but skip the editor & runner (I could not, at least not well).

Evidently I forgot to clean it up.